### PR TITLE
Create ViewerAddCommand to add pc of a given name to the list of pc

### LINKED ISSUE #12: Viewer [add <pcname>]
Adds a new pc with that name to the PClist

### PR DESCRIPTION
**Pull request by Reynold-SL**
Resolves #12 
Create a command to add pc of a given name to the list of pc

### CODE REVIEW

File: src/main/java/seedu/bigpp/parser/Parser.java [hunk 1]
```diff
@@ -57,11 +59,15 @@ private Command parseMainMenuCommand(String userInput) {
 
     private Command parseViewerCommand(String userInput) {
         String commandWord = userInput.split(" ")[0];
+        String arguments = userInput.split(" ")[1];
```
Comment (Ryujikjs) [inline src/main/java/seedu/bigpp/parser/Parser.java:62]: Can try putting this line below instead in the case: "add" as other commands such as "bye"(to be implemented in the future does not have an argument. Should handle exceptions for this case.

File: src/main/java/seedu/bigpp/command/viewercommand/ViewerAddCommand.java [hunk 2]
```diff
@@ -1,12 +1,20 @@
...
+    }
+
     @Override
     public String executeCommand() {
-        // TODO Auto-generated method stub
-        return "";
+        String name = super.getArguments();
+        PC pc = new PC(name, false);
+        PCList.addPC(pc);
+        return "I have created the PC with the following name: " + name;
```
Comment (Ryujikjs) [inline src/main/java/seedu/bigpp/command/viewercommand/ViewerAddCommand.java:18]: Maybe can use statement such as "Custom PC: [ name ] has been created" to follow format of pc viewer

**Review by Ryujikjs — APPROVED** [review]
Overall code looks good for v0.1 implementation, just make sure to handle exceptions in the future.

File: src/main/java/seedu/bigpp/command/viewercommand/ViewerAddCommand.java (hunk 2 above)
Comment (Magmanat) [reply src/main/java/seedu/bigpp/command/viewercommand/ViewerAddCommand.java:18]: agreed, this change sounds good

File: src/main/java/seedu/bigpp/parser/Parser.java (hunk 1 above)
Comment (Reynold-SL) [reply src/main/java/seedu/bigpp/parser/Parser.java:62]: #34 Create issue and will solve in future iterations

**Pull request merged**